hipchat.go

```diff
@@ -218,4 +225,5 @@ func (c *Client) listen() {
 			}
 		}
 	}
+	panic("unreachable")
```

Comment:
Dane Harrigan <notifications@github.com> wrote:
> > @@ -218,4 +225,5 @@ func (c *Client) listen() {
> >  			}
> >  		}
> >  	}
> > +	panic("unreachable")
> 
> I don't understand why the `panic` was added. just in case something unexpected happens?

Yeah.  This could be dropped and is not related to the priv message
functionality.